### PR TITLE
Add ipymidicontrols

### PR DESCRIPTION
**Pull request by jasongrout**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jasongrout** [comment]
CC @SylvainCorlay - are you okay also being a maintainer for this?

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,82 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
-
```
Comment (scopatz) [inline recipes/example/meta.yaml]: please restore the example

Comment (jasongrout) [reply recipes/example/meta.yaml]: Thanks. I was confused by the directions, and wasn't sure what the end goal of the PR was.

**Comment by jasongrout** [comment]
Do force pushes not trigger the CI? I was hoping to not introduce a spurious change in the example by force-pushing a fix.

**Comment by jasongrout** [comment]
@msarahan - it seems that the conda 4.7.6 or conda-build 3.17.8 upgrade is causing errors on all recent conda-forge builds: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=49363&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=256&lineEnd=274&colStart=1&colEnd=62

```
+ xargs -n1 cat
+ xargs -r /usr/bin/sudo -n yum install -y
+ find /home/conda/conda-recipes -mindepth 2 -maxdepth 2 -type f -name yum_requirements.txt
+ grep -v -e '^#' -e '^$'
+ test 1 == 1
+ python /home/conda/.ci_support/build_all.py /home/conda/conda-recipes
Building ipymidicontrols with conda-forge/label/main
Traceback (most recent call last):
  File "/home/conda/.ci_support/build_all.py", line 131, in <module>
    build_all(args.recipes_dir, args.arch)
  File "/home/conda/.ci_support/build_all.py", line 68, in build_all
    build_folders(recipes_dir, new_comp_folders, arch, channel_urls)
  File "/home/conda/.ci_support/build_all.py", line 94, in build_folders
    conda_resolve = conda_build.conda_interface.Resolve(index)
  File "/opt/conda/lib/python3.7/site-packages/conda/resolve.py", line 116, in __init__
    self._system_precs = {_ for _ in index if _.package_type == PackageType.VIRTUAL_SYSTEM}
  File "/opt/conda/lib/python3.7/site-packages/conda/resolve.py", line 116, in <setcomp>
    self._system_precs = {_ for _ in index if _.package_type == PackageType.VIRTUAL_SYSTEM}
AttributeError: 'Dist' object has no attribute 'package_type'
```

Some examples of builds having this error, all having conda 4.7.6, conda-build 3.18.7:

* https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=49404&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=183&lineEnd=185&colStart=1&colEnd=1
* https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=49419&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=183&lineEnd=185&colStart=1&colEnd=1
* https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=49420&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=183&lineEnd=185&colStart=1&colEnd=1

Examples of successful builds just before these, all having conda 4.7.5, conda-build 3.17.8:
* https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=49197&view=logs&jobId=240f1fee-52bc-5498-a14a-8361bde76ba0&taskId=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&lineStart=99&lineEnd=101&colStart=1&colEnd=1
* https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=49327&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=183&lineEnd=185&colStart=1&colEnd=1
* https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=49127&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=183&lineEnd=185&colStart=1&colEnd=1

**Comment by jasongrout** [comment]
I opened https://github.com/conda/conda/issues/8916 about this issue.

CC @mwcraig @jakirkham @scopatz

File: recipes/ipymidicontrols/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ hash }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed --no-cache-dir"
```
Comment (xhochy) [inline recipes/ipymidicontrols/meta.yaml]: No need for the arguments, they are all included by default now.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```

Comment (jasongrout) [reply recipes/ipymidicontrols/meta.yaml]: Thanks. I was wondering about that.

**Pull request merged**